influence_benchmark/config/accelerate_config.py

```diff
@@ -24,8 +26,13 @@ def set_gpu_ids(self, gpu_ids: Optional[List[int]]):
             return
         # NOTE: Currently only support one GPU, maybe this is not what we want for KTO?
         max_gpus = 1
-        self.gpu_ids = gpu_ids[:max_gpus]
+        if len(gpu_ids) > max_gpus:
+            gpu_ids = np.random.choice(gpu_ids, max_gpus, replace=False).tolist()
```

Comment:
seems quite convoluted to do a random gpu? we could just do the first one?

Reply:
Yeah i had a reason I wanted to do random, but turned out to not actually help much at all. Will revert